README.md

```diff
@@ -112,6 +112,39 @@ title,content
 My Title,Hello World!
 ```
 
+### Processing Batch Collections
+
+The Maintenance Tasks gem supports processing Active Records in batches. This
+can reduce the number of calls your Task makes to the database. Use
+`ActiveRecord::Batches#in_batches` to specify that your Task should process
+records in batches. The default batch size is 1000, but a custom size can be
+specified.
+
+```ruby
+# app/tasks/maintenance/update_posts_in_batches_task.rb
+module Maintenance
+  class UpdatePostsInBatchesTask < MaintenanceTasks::Task
+    def collection
+      Post.in_batches
+    end
+
+    def count
+      Post.count
+    end
+
+    def process(batch_of_posts)
+      batch_of_posts.update_all(content: "New content added on #{Time.now.utc}")    
+    end
+  end
+end
+```
+
+Ensure that you've implemented the following methods:
+* `collection`: return an `ActiveRecord::Batches::BatchEnumerator`.
+* `process`: do the work of your Task on a batch (`ActiveRecord::Relation`).
+* `count`: return the number of rows that will be iterated over (this should
+still be the number of records to be processed, not the number of batches).
```

Comment:
Is it? Why not use the number of batches instead? I guess showing the number of records does look nicer.